packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -82,6 +83,27 @@ const parseOpenAPIParams = (params: OpenAPIParamsType[]): HoppRESTParam[] =>
     )
   )
 
+const parseOpenAPIVariables = (
+  variables: OpenAPIParamsType[]
+): HoppRESTRequestVariable[] =>
+  pipe(
+    variables,
+
+    A.filterMap(
+      flow(
+        O.fromPredicate((param) => param.in === "path"),
+        O.map(
+          (param) =>
+            <HoppRESTRequestVariable>{
+              key: param.name,
+              value: "", // TODO: Can we do anything more ? (parse default values maybe)
```

Comment:
`value` won't be present in the incoming data, right? Are we keeping it empty so that filling up the value is up to the user?

Reply:
yup the value will be empty while importing